src/thunderbots/software/util/parameter/config/ai.yaml

```diff
@@ -83,6 +83,14 @@ AI:
         Maximum time into the future at which the pass should occur. This gives the
         upper bound on the pass start time, relative to the current time. This is in
         seconds.
+    enemy_intercept_delay:
```

Comment:
I think "reaction time" would be better here, and it matches what's in the code.

Reply:
Great point, changed.